app/boards/intel_adsp_ace15_mtpm.conf

```diff
@@ -11,6 +11,7 @@ CONFIG_COMP_CHAIN_DMA=y
 CONFIG_COMP_CROSSOVER=y
 CONFIG_COMP_DRC=y
```

Comment:
Does this work for IPC3?

Reply:
@dbaluta no
Will add condition to kconfig